app/javascript/src/Users/permissions.jsx

```diff
@@ -229,7 +229,7 @@ export const Form = { // eslint-disable-line no-unused-vars
       partners: 'Developer Accounts -- Applications',
       monitoring: 'Analytics',
       plans: 'Integration & Application Plans',
-      policy_registry: 'Policy Registry'
+      policies_registry: 'Policies Registry'
```

Comment:
I was wondering if we could use locales later. @josemigallas @damianpm @thomasmaas  @didierofrivia do you think it is possible

Reply:
Permissions are being refactored here https://github.com/3scale/porta/pull/604, we include localization as well there.